xbmc/video/dialogs/GUIDialogVideoInfo.cpp

```diff
@@ -1582,9 +1582,10 @@ bool CGUIDialogVideoInfo::GetMoviesForSet(const CFileItem *setItem, CFileItemLis
   if (!videodb.Open())
     return false;
 
-  std::string baseDir = StringUtils::Format("videodb://movies/sets/%d", setItem->GetVideoInfoTag()->m_iDbId);
+  std::string dir =
```

Comment:
I know that a previous comment from @enen92 suggested you rename the variable to just dir, but the final implementation you have  makes the variable name change the only change in this unit. Hence why do it? For me it just makes it hard to see what you have done.

Reply:
He did it on a second commit, actual logic changes are in https://github.com/xbmc/xbmc/pull/17865/commits/7c738a9dc64d6e062669b7269426e6c2c8ff0f88

Reply:
Fair enough, but I wouldn't change a var name at all in this kind of work it really does not make anything less confusing. `basDir` is commonly used as a var name for similar purposes thoughout the older code, maybe I am just familiar with it.

Reply:
as i said, a matter of taste. imo we can stay with `baseDir` i just wanted to take the opportunity due to the suggestion. as the whole thing is not yet 100%-ready (checking a 2nd dialog) we can decide what to do with the cleanup commit. to me it makes no big difference (apart from a confusing diff)

Reply:
Yes, sorry @howie-f that you are piggy-in-the-middle here over a var name discussion. Total madness :)

Reply:
np, decided to cut off